learning/tour-of-beam/learning-content/go/common-transforms/filter/description.md

```diff
@@ -0,0 +1,34 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Filter
+
+### Using Filter
+
+PCollection datasets can be filtered using the Filter transform. You can create a filter by supplying a predicate and, when applied, filtering out all the elements of PCollection that don’t satisfy the predicate.
+
+```
+import (
+	"github.com/apache/fbeam/sdks/go/pkg/beam"
+	"github.com/apache/beam/sdks/go/pkg/beam/transforms/filter"
+)
+
+func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
+	return filter.Exclude(s, input, func(element int) bool {
+		return element % 2 == 1
+	})
+}
+```
+
+### Description for example
+
+Given a list of integers , printing even numbers using ```Filter```. The ```applyTransform()``` function implements a filter in which the logic determines the numbers are even.
```

Comment:
Same as previous comments - add an invitation to run the example in the playground window, a small challenge may be working with strings